CHANGELOG.md

```diff
@@ -346,3 +346,5 @@ Released with 1.0.0-beta.37 code base.
 
 1. `create` function does not take in the optional parameter `entropy`
 2. `ignoreLength` will be removed as an optional parameter for `privateKeyToAccount`
+3. `signTransaction` will not fill any default values and it will only sign and return result. For filling default values use web3.transaction package
```

Comment:
```suggestion
3. `signTransaction` will not fill any default values and it will only sign and return result. For filling default values use `web3-eth-transaction` package
```